octoprint_Spoolman/templates/Spoolman_sidebar.jinja2

```diff
@@ -127,7 +130,10 @@
                             <span>[<span data-bind="
                                 text: $data.spoolDisplayData.filament.material.displayShort,
                                 attr: { title: $data.spoolDisplayData.filament.material.displayValue }
-                            "></span>]</span>
+                            "></span>] <span data-bind="
```

Comment:
Putting the lot number before the "title" (name & brand of the filament) seems odd to me. Similar to the id proposal from the previous comment, I think it would be better to put it below, either on the same line as id, or one line below.

Reply:
The decision here was based on the particular filament I was testing with preforming a line break after the filament type, I thought there may have been a purpose for this, and so thought there would be a good place to fill some unused space, however I do agree that below all of this information is a much better suited place for this. I will explore this in the other PR.